frontend/src/services/location/locationService.js

```diff
@@ -27,6 +27,20 @@ export class LocationService {
         });
     }
 
+    getCityByLocation(mapboxToken, lng, lat){
+        return new Promise((resolve, reject) => {
+            let mapboxCitiesApiUrl = `https://api.mapbox.com/geocoding/v5/mapbox.places/${lng},${lat}.json?access_token=${mapboxToken}&country=ua&types=place&language=en`;
```

Comment:
https://api.mapbox.com/geocoding/v5/mapbox.places/ - const var.  create method is mapSettingsService createCitiesApiUrl(query)